modules/utils/tests/string_literal_tests.cpp

```diff
@@ -0,0 +1,49 @@
+//=================================================================================================
+// Copyright (C) 2023-2024 HEPHAESTUS Contributors
+//=================================================================================================
+
+#include <random>
+
+#include <gtest/gtest.h>
+
+#include "hephaestus/utils/string/string_literal.h"
+
+// NOLINTNEXTLINE(google-build-using-namespace)
+using namespace ::testing;
+
+namespace heph::utils::string::tests {
+
+/// Generate a random string of characters, including special case characters and numbers.
+auto randomString(std::mt19937_64& mt) -> std::string {
```

Comment:
this feels like a duplicate from the other PR, maybe you can extract it into a common file in the `test` folder

Reply:
This was a mistake, it's actually not used. Removed.